### ISSUE TITLE
[JOSE review]

### ISSUE THREAD

**Issue by darribas** [martibosch/swisslandstats-geopy#11]
A few comments on the paper:

- [x] Statement of need: I think whether there's a subsection or not, it'd be helpful to start the paper with a (short) paragraph on why the package is required and what problem it solves. 
- [x] The point above could also be included in the `README.md` file on top, just before the target audience.
- [x] To make the paper a bit more useful for the reader and prove its point more effectively, I think it'd help to provide short code snippets on the list of features listed at the end of the *Summary* section. I'm thinking, for every bullet point, to include a "demo snippet". For example, for the first one, it could read:

---
> * Read CSV files from the SFSO intoLandDataFrameobjects, which extend the conven-tional pandasDataFramewith additional attributes that store the coordinate referencesystem (CRS) and pixel resolution.

```python
import swisslandstats
ldf = swisslandstats.read_csv('data/AREA_NOAS04_17_181029.csv')
```
---

- [ ] Finally, I think the functionality in this paper is super cool and interesting and I wonder whether the package could be used in other contexts where data is gridded in a similar way (is there any? I know there's a lot of gridded datasets that are provided as text files but I'm not sure if the format is the same). It'd be great to elaborate a bit more on this, either stating other datasets for which this is relevant or, maybe more useful, discussing what are the characteristics that a dataset needs to fulfill. 

[openjournals/joss-reviews#1511](https://github.com/openjournals/joss-reviews/issues/1511)

**Comment by martibosch** [comment]
Hello,

> * [ ]  Statement of need: I think whether there's a subsection or not, it'd be helpful to start the paper with a (short) paragraph on why the package is required and what problem it solves.
> * [ ]  The point above could also be included in the `README.md` file on top, just before the target audience.

I have added the statement of need both in the paper and the readme in [8a7d0ab](https://github.com/martibosch/swisslandstats-geopy/commit/8a7d0abb8dc02ce8ef108dbe80ef8e1009bbf8ef)

> * [ ]  To make the paper a bit more useful for the reader and prove its point more effectively, I think it'd help to provide short code snippets on the list of features listed at the end of the _Summary_ section. I'm thinking, for every bullet point, to include a "demo snippet". For example, for the first one, it could read:

I have added a demo snippet for each feature in the paper in [a8f34fe](https://github.com/martibosch/swisslandstats-geopy/commit/a8f34fefb30ef93cf6b099556d14864b45321ff5)

> * [ ]  Finally, I think the functionality in this paper is super cool and interesting and I wonder whether the package could be used in other contexts where data is gridded in a similar way (is there any? I know there's a lot of gridded datasets that are provided as text files but I'm not sure if the format is the same). It'd be great to elaborate a bit more on this, either stating other datasets for which this is relevant or, maybe more useful, discussing what are the characteristics that a dataset needs to fulfill.

I actually surveyed the reusability of the package as well (I even thought of calling it "landas").
Nevertheless, I could only find [one similarly-structured dataset](https://daac.ornl.gov/VEGETATION/guides/Phenology_Deciduous_Forest.html) which could be seemingly converted to a ndarray with this library out of the box. I could not do the same with other table-like land datasets, e.g., [LUCAS](https://ec.europa.eu/eurostat/statistics-explained/index.php/LUCAS_-_Land_use_and_land_cover_survey), and I believe that is because the CRS (e.g., LUCAS uses WGS84) and its respective float numbers impede a row-to-pixel matching in [the `LandDataFrame.to_ndarray`](https://github.com/martibosch/swisslandstats-geopy/blob/master/swisslandstats/dataframe.py#L101-L103) method. Instead, the LV95/LV03 used by the SFSO consists of integers, always separated by 100, which are forthwith mapped to a pixel.

To summarize, I believe there might be some reusability potential which unluckily I do not have the time to address properly. However, I agree that it could be worth a small paragraph in the paper. Should I?

**Comment by darribas** [comment]
Excellent. Yes, I think a couple of lines reflecting on what it'd take to make this more general would be a great addition!

**Comment by martibosch** [comment]
I have included a dedicated paragraph with references to the datasets in [d1a9913](https://github.com/martibosch/swisslandstats-geopy/commit/d1a991352bd46054c8c6aea43eb0566776ca441a). It is 11 lines (a bit more than "a couple"), but I could make it shorter if needed.

**Comment by martibosch** [comment]
I found out yesterday that [there are more geo-datasets provided by the SFSO](https://www.bfs.admin.ch/bfs/en/home/services/geostat/swiss-federal-statistics-geodata.html) that make use of the same table-like format. Had I known about this, I would have chosen another title for the library - but well, since we're still talking about geospatial data, the "/landstats" part is not too misleading either.

In any case, I have made some amendments, released them in v0.8.0 and added a section that shows how the library can also be used with the Swiss population statistics (STATPOP).
